airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -246,8 +246,8 @@
               "plugin": {
                 "type": "string",
                 "title": "Plugin",
-                "description": "A logical decoding plugin installed on the PostgreSQL server. The `pgoutput` plugin is used by default. The `wal2json` plugin is deprecated and will soon be removed so it's not recommended to use. Read more about <a href=\"https://docs.airbyte.com/integrations/sources/postgres#step-2-select-a-replication-plugin\">selecting replication plugins</a>.",
-                "enum": ["pgoutput", "wal2json"],
+                "description": "A logical decoding plugin installed on the PostgreSQL server.",
+                "enum": ["pgoutput"],
                 "default": "pgoutput",
                 "order": 2
               },
```

Comment:
Should we just remove this?
So far there is only one option, and showing it to the users would be confusing.
```suggestion
```

Reply:
I was thinking the same but while testing I realised that the front end is smart enough to not display the option if there is only 1 option available and default is present so I let it be for now since I didn't want to change a lot. In future once this version is running in prod safely then we can get rid of it. 
<img width="669" alt="Screenshot 2023-02-27 at 10 20 47 PM" src="https://user-images.githubusercontent.com/28527634/221627487-36b70a0d-deaf-477a-8ec5-9a51b3ced2d7.png">

Reply:
Nice!
Btw, if there is no more plugins and we don't support them, I would just remove it, less code is better code.

Reply:
If we remove this from the spec then the config of all the existing connectors would needed to be migrated as well which would increase the scope. I will create an issue to clean this up as part of refactoring work